doc/send_request.md

```diff
@@ -0,0 +1,269 @@
+# Using your Client's `send_request` Method
+
+In this doc, we will be showing how to build your own ***http requests*** and how to send these requests directly to the service using the ***send_request*** method.
+
+Here's how to get started:
+
+```python
+>>> from azure.identity import DefaultAzureCredential
+>>> from azure.example.service import ExampleClient
+>>> from azure.core.rest import HttpRequest, HttpResponse
+>>> client = ExampleClient(endpoint="https://www.example.org/", credential=DefaultAzureCredential())
+>>> request = HttpRequest(method="GET", url="https://www.example.org")
+>>> request
+<HttpRequest [GET], url: "https://www.example.org">
+>>> response = client.send_request(request)
+>>> response
+<HttpResponse: 200 OK, Content-Type: text/plain>
+>>> response.raise_for_status()
+>>> response.text()
+'Happy to see you!'
+```
+
+## Code Snippets
+
+**End-to-end code snippets for creating and sending requests with `send_request`**:
+
+1. [Sync client](#sync-client)
+2. [Async client](#async-client)
+
+## Steps To Make a Call
+
+1. [Initialize your client](#1-initialize-your-client "Initialize Your Client")
+2. [Create a request](#2-create-a-request "Create a Request")
+3. [Send the request](#3-send-the-request "Send the Request")
+4. [Handle the response](#4-handle-the-response "Handle the Response")
+
+We will go into each step in the following sections
+
+## 1. Initialize Your Client
+
+First you import your client from the namespace of your package. For example, let's say your namespace is `azure.example.service` and your client's name
+is `ExampleClient`. Your import would look like
+
+```python
+from azure.example.service import ExampleClient
+```
+
+Most clients require authenticating through their `credential` parameter. Depending on what authentication support your library is using, you can either [authenticate with aad](#authenticating-with-aad) or [authenticate with an `AzureKeyCredential`](#authenticating-with-azurekeycredential).
+
+Additionally, most of our clients accept an `endpoint` parameter at initialization, usually a link to your own resource.
+
+### Authenticating with AAD
```

Comment:
Do we need this at all? Shouldn't this document really focus on send_request, and then assume you have a client already?